### PR TITLE
feat: add edit mode markups to CollectorServiceAccountsSection

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

https://github.com/cloudforet-io/console/assets/26986739/e5530b22-079b-4434-899b-2747b0aad356

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue [hunk 1]
```diff
@@ -58,11 +59,13 @@ import { useCollectorFormStore } from '@/services/asset-inventory/store/collecto
 
 interface Props {
     title?: TranslateResult;
+    marginOnSpecific?: boolean;
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue:62]: 😮

File: apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue [hunk 2]
```diff
@@ -79,7 +82,7 @@ const attachedServiceAccountList = [
 ];
 
 const state = reactive({
-    selectedAttachedServiceAccountType: 'all' as 'all'|'specific',
+    selectedAttachedServiceAccountType: 'all' as SelectType,
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue:85]: praise: 👍

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue [hunk 3]
```diff
@@ -147,6 +150,10 @@ watch(() => isAllValid.value, (value) => {
 
 <style lang="postcss" scoped>
 .attached-service-account-form {
+    .margin-on-specific {
+        margin-bottom: 15rem;
+    }
+
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/modules/AttachedServiceAccountForm.vue:156]: I agree with your opinion 😥

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**